trento/migration/systemd-install.md

```diff
@@ -430,7 +432,7 @@ This is a basic guide for creating a self-signed certificate for use with Trento
        listen 443 ssl;
        server_name trento.example.com;
 
-       ssl_certificate /etc/ssl/certs/trento.crt;
+       ssl_certificate /etc/nginx/ssl/certs;
```

Comment:
`ssl_certificate /etc/nginx/ssl/certs/trento.crt;  `

We should be specific here